tests/non-js-resources/subtests/dependencies/subtests/deduping/webpack/webpack.config.js

```diff
@@ -15,12 +15,11 @@ module.exports = {
     rules: [
       {
         test: /\.css$/,
-        enforce: 'pre',
-        use: ['extract-loader', 'css-loader'],
+        use: ['style-loader', 'css-loader'],
       },
       {
-        test: /\.(css|ttf|png)$/,
-        loader: 'file-loader',
+        test: /\.(ttf|png)$/,
+        type: 'asset/resource',
       },
     ],
   },
```

Comment:
```suggestion
  },
  plugins: [new (require('mini-css-extract-plugin'))()],
```

(would prefer to hoist the require)